rules/php73/tests/Rector/FuncCall/RemoveMissingCompactVariableRector/Fixture/compact_over_array.php.inc

```diff
@@ -2,14 +2,31 @@
 
 namespace Rector\Php73\Tests\Rector\FuncCall\RemoveMissingCompactVariableRector\Fixture;
 
-function run()
+class CompactOverArray
 {
-    $foo = 'foo';
-    $bar = 'bar';
+    private function test_argument_without_existing_vars_will_be_removed()
```

Comment:
These names are already in fixture file name.

![image](https://user-images.githubusercontent.com/924196/91206467-7c096800-e707-11ea-8556-ffbbba24ff42.png)


The convention is one fixture = one case. That way we have minimal code that if broken, easy easy to fix.

Saying that, could you split it to 3 different files?

Reply:
Done